python/cudf/cudf/core/series.py

```diff
@@ -3669,6 +3669,78 @@ def merge(
 
         return result
 
+    def add_prefix(self, prefix):
+        """
+        Prefix labels with string `prefix`.
+        For Series, the row labels are prefixed.
+        Parameters
+        ----------
+        prefix : str
+            The string to add before each label.
+        Returns
+        -------
+        Series
+            New Series with updated labels.
+        See Also
+        --------
+        Series.add_suffix: Suffix row labels with string `suffix`.
+        Examples
+        --------
+        >>> s = cudf.Series([1, 2, 3, 4])
+        >>> s
+        0    1
+        1    2
+        2    3
+        3    4
+        dtype: int64
+        >>> s.add_prefix('item_')
+        item_0    1
+        item_1    2
+        item_2    3
+        item_3    4
+        dtype: int64
+        """
+        result = self.copy(deep=False)
+        result = result.set_index(prefix + self._index.astype(str))
```

Comment:
`Series.set_index` is deprecated, perhaps a more long-term way to do this would be:

```suggestion
        result.index = prefix + self.index.astype(str)
```